yarn.lock

```diff
@@ -2484,11 +2484,6 @@ anymatch@~3.1.1:
     normalize-path "^3.0.0"
     picomatch "^2.0.4"
 
-app-module-path@^2.2.0:
```

Comment:
Not sure why all of this got deleted... still getting used to yarn.

Reply:
Opps, probably happened when we last merged main. Once the project matures, the dependencies will stabilize.